src/components/__tests__/__snapshots__/title.test.js.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is it necessary to keep snapshots in the repository?

Reply:
I researched about the same before committing. As I've said above, [Jest ~recommends~ insists on keeping them](https://jestjs.io/docs/en/snapshot-testing.html#should-snapshot-files-be-committed).

My understanding: Snapshots are required by Jest so that it can match the generated tree with them. If not available, snapshots are generated again by Jest for running tests. **This can slow down the tests when you run them for the first time** in your machine.

Bottom line is, having snapshots in the repo can increase the pace of testing, for new users. So my opinion is to keep them if we plan to implement more tests.

Reply:
Oh Cool. Then we can keep it.